## 02_fix_pkg-config_paths.dpatch by Mirco Bauer <meebey@debian.org>
##
## Modified by Archlinux: removed libdir change

diff -urNad gnome-sharp-2.24.1~/art/art-sharp-2.0.pc.in gnome-sharp-2.24.1/art/art-sharp-2.0.pc.in
--- gnome-sharp-2.24.1~/art/art-sharp-2.0.pc.in	2009-03-21 19:02:54.000000000 +0100
+++ gnome-sharp-2.24.1/art/art-sharp-2.0.pc.in	2009-03-21 19:03:32.000000000 +0100
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
diff -urNad gnome-sharp-2.24.1~/gconf/GConf/gconf-sharp-2.0.pc.in gnome-sharp-2.24.1/gconf/GConf/gconf-sharp-2.0.pc.in
--- gnome-sharp-2.24.1~/gconf/GConf/gconf-sharp-2.0.pc.in	2009-03-21 19:02:54.000000000 +0100
+++ gnome-sharp-2.24.1/gconf/GConf/gconf-sharp-2.0.pc.in	2009-03-21 19:03:26.000000000 +0100
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 
diff -urNad gnome-sharp-2.24.1~/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in gnome-sharp-2.24.1/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in
--- gnome-sharp-2.24.1~/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in	2009-03-21 19:02:54.000000000 +0100
+++ gnome-sharp-2.24.1/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in	2009-03-21 19:03:20.000000000 +0100
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 
diff -urNad gnome-sharp-2.24.1~/gnome/gnome-sharp-2.0.pc.in gnome-sharp-2.24.1/gnome/gnome-sharp-2.0.pc.in
--- gnome-sharp-2.24.1~/gnome/gnome-sharp-2.0.pc.in	2009-03-21 19:02:54.000000000 +0100
+++ gnome-sharp-2.24.1/gnome/gnome-sharp-2.0.pc.in	2009-03-21 19:03:07.000000000 +0100
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
diff -urNad gnome-sharp-2.24.1~/gnomevfs/gnome-vfs-sharp-2.0.pc.in gnome-sharp-2.24.1/gnomevfs/gnome-vfs-sharp-2.0.pc.in
--- gnome-sharp-2.24.1~/gnomevfs/gnome-vfs-sharp-2.0.pc.in	2009-03-21 19:02:54.000000000 +0100
+++ gnome-sharp-2.24.1/gnomevfs/gnome-vfs-sharp-2.0.pc.in	2009-03-21 19:03:12.000000000 +0100
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
